utils/certs.go

```diff
@@ -93,7 +93,7 @@ func GenerateCert(hosts []string, certFile, keyFile, caFile, caKeyFile, org stri
 	}
 	// client
 	if len(hosts) == 1 && hosts[0] == "" {
-		template.ExtKeyUsage = []x509.ExtKeyUsage{x509.ExtKeyUsageClientAuth}
+		template.ExtKeyUsage = []x509.ExtKeyUsage{x509.ExtKeyUsageClientAuth, x509.ExtKeyUsageServerAuth}
```

Comment:
Setting this for ServerAuth here unfortunately doesn't do much since we aren't setting any hosts for it to be valid for.
If you try to use it for ServerAuth you end up with an error like so: `cannot validate certificate for 127.0.0.1 because it doesn't contain any IP SANs` or `certificate is valid for , not localhost`